src/components/BulkActionsToolbar/BulkActionsToolbar.tsx

```diff
@@ -25,6 +25,7 @@ export type BulkActionsToolbarAction = {
     label: string;
     icon?: React.ReactElement;
     tooltip?: string;
+    delay?: number;
```

Comment:
What do you think about renaming the prop to `tooltipDelay`?

Reply:
yes